index.html

```diff
@@ -388,9 +388,101 @@ <h2 id="protocol-registry">
     <h3>
       Inclusion criteria
     </h3>
+    <aside class="note">
+      The below criteria are a work in progress and are likely to change as
+      this document evolves.
+    </aside>
+    <p>
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
+    </p>
+    <ol>
+      <li>MUST be standardized at a recognized standards organization and have
+      a stable URL.
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation of the
```

Comment:
Is it worth including a non-normative example?

Reply:
Certainly could, but if we can get one into OpenID, then that can serve as a good basis. Alternatively, we are going to specify that fake test protocol... so we could point to that too as an example.